node/runtime/src/lib.rs

```diff
@@ -59,7 +59,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
 	spec_version: 90,
```

Comment:
doesn't this change spec version ?

Reply:
I meant to bump `spec_version`, but it got lost in a merge conflict resolution.